### DISCUSSION TITLE
blog/demo/markdown

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#517] [Announcements]
# blog/demo/markdown

VuePress 主要从 Markdown 文件生成页面。因此，你可以使用它轻松生成文档或博客站点。 你应该创建和编写 Markdown 文件，以便 VuePress 可以根据文件结构将它们转换为不同的页面。

http://localhost:8081/blog/demo/markdown.html

**Comment by ynushehao** [comment]
你好
